star-wars-browser/src/components/search_results/SearchResults.jsx

```diff
@@ -3,48 +3,58 @@ import PropTypes from 'prop-types';
 import SingleResult from '../single_result';
 import Paginate from './Paginate';
 
-const SearchResults = ({ pageNumber, changePage, peopleData }) => {
+const SearchResults = ({
+  pageNumber,
+  changePage,
+  people,
+}) => {
   const [labelClassName, setLabelClassName] = useState(
     'search-results-label__list d-none',
   );
 
-  const currentCharactersPage = peopleData[pageNumber - 1];
+  const currentCharactersPage = people[pageNumber - 1];
```

Comment:
jak wygląda zmienna people? ona zawiera po prostu obiekty postaci, czy tablicę stron z postaciami?

Reply:
j.w. people to array klastrów ( jest ich tyle ile stron ), po 10 postaci w klastrze